Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -894,6 +894,9 @@ private void HandleOrderEvent(OrderEvent fill)
             // set the status of our order object based on the fill event
             order.Status = fill.Status;
 
+            // set the modified time of the order to the fill's timestamp
+            order.ModifiedTime = fill.UtcTime;
```

Comment:
Isn't this called on every order status change including invalid, cancel ?
Maybe should be called `LastEventTime` ?

Reply:
agreed -- ended u splitting into three separate properties